packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -54,15 +64,19 @@ describe('WorkspaceContext', () => {
   it('test for the constructor', () => {
     expect(workspaceContextUtil).toHaveProperty('sessionConnections');
     expect(workspaceContextUtil).toHaveProperty('onOrgChangeEmitter');
-    expect(workspaceContextUtil).toHaveProperty('cliConfigWatcher', mockWatcher);
+    expect(workspaceContextUtil).toHaveProperty(
+      'cliConfigWatcher',
+      mockWatcher
+    );
   });
-  it('should load the default username and alias upon initialization', () => {
+  it('should load the default username and alias and clear the cache of the core types upon initialization', () => {
     expect(workspaceContextUtil.username).toEqual(testUser);
     expect(workspaceContextUtil.alias).toEqual(testAlias);
     expect(reloadConfigAggregatorsMock).toHaveBeenCalled();
+    expect(stateAggregatorClearInstanceMock).toHaveBeenCalled();
   });
 
-  it('should update default username and alias upon config change', async () => {
+  it('should update default username and alias and clear the cache of the core types upon config change', async () => {
```

Comment:
nit: Just an observation that if a function always does a thing (like clearing the cache) I don't usually call it out in the test description.

Reply:
nah it's fine.  I was just imagining updating the test description with every update to a method and could see these getting huge. no biggie